core/zstd/src/ZipZSTD.cxx

```diff
@@ -24,11 +24,22 @@ static const size_t errorCodeSmallBuffer = (size_t)-70;
 
 void R__zipZSTD(int cxlevel, int *srcsize, char *src, int *tgtsize, char *tgt, int *irep)
 {
+
+    if (*tgtsize <= 0) {
```

Comment:
```suggestion
    if (*tgtsize <= kHeaderSize) {
```

Reply:
This is validated at a higher level:
https://github.com/root-project/root/blob/239cafe11e41a862c35e1f9d0e120cc2a73a3654/core/zip/src/RZip.cxx#L87-L90
(introduced by https://github.com/root-project/root/commit/23261a631c9e53a2ca7981a61d174c6465873a0e; in general, there was some work in this area in the context of https://github.com/root-project/root/pull/14523)